controllers/suite_test.go

```diff
@@ -57,6 +59,11 @@ var _ = BeforeSuite(func() {
 		CRDDirectoryPaths: []string{
 			filepath.Join("..", "config", "crd", "bases"),
 		},
+		// Add fake CAPI CRDs that we reference
+		CRDs: []*apiextensionsv1.CustomResourceDefinition{
+			external.TestClusterCRD.DeepCopy(),
```

Comment:
What is the benefit of faking the CRDs instead of including the actual ones? (I have not really worked with envtest before)

Reply:
I think the main benefit is that we don't have to include the CAPI CRDs either in the repo or automatically download them to be able to run the tests.

Reply:
Ah right, because this expects `CustomResourceDefinition` objects, but we currently only import the API types (from `sigs.k8s.io/cluster-api/api/*`.

That makes sense. Thank you :)